files/sanity_check.py

```diff
@@ -49,13 +49,10 @@ def check_port(host, port, retries=DEFAULT_PORT_RETRIES, retry_delay=DEFAULT_POR
 
 def check_command(command):
     try:
-        devnull = open(os.devnull, 'w')
-        subprocess.check_call(
-            command, shell=True, stdout=devnull, stderr=subprocess.STDOUT
-        )
-        return True
-    except subprocess.CalledProcessError:
-        return False
+        output = subprocess.check_output(command, shell=True, stderr=subprocess.STDOUT)
```

Comment:
@UmanShahzad Might make sense to rename to `run_command` or something similar - it's not just checking if command execution succeeded, but also return output.

Reply:
@e-kolpakov You mean the `check_command` function is to change, right? I'll confirm first before making changes, but that sounds good to me to change.

Reply:
@UmanShahzad yes `check_command` - > `run_command`

Reply:
:+1: